tutorial/source/enumeration.ipynb

```diff
@@ -207,6 +210,49 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Examining discrete latent states <a class=\"anchor\" id=\"Examining-discrete-latent-states\"></a>\n",
+    "\n",
+    "While enumeration in SVI allows fast learning of parameters like `p` above, it does not give access to predicted values of the discrete latent variables like `x,y,z` above. We can access these using a standalone `infer_discrete` poutine. In this case the guide was trivial, so we can simply wrap the model in `infer_discrete`. We need to pass a `first_available_dim` argument to tell `infer_discrete` which dimensions are available for enumeration; this is related to the `max_plate_nesting` arg of `TraceEnum_ELBO` via\n",
```

Comment:
Can you mention that score-function gradients through `infer_discrete` won't be correct? Also a nit: "standalone `infer_discrete` poutine" -> "standalone `infer_discrete` handler"